api/v1beta1/nmstate_types.go

```diff
@@ -34,6 +34,14 @@ type NMStateSpec struct {
 	// If Tolerations is specified, the handler daemonset will be also scheduled on nodes with corresponding taints
 	// +optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+	// WebhookNodeSelector is an optional selector that will be added to webhook & certmanager Deployment manifests
+	// If WebhookNodeSelector is specified, the webhook and certmanager will run only on nodes that have each of the indicated
+	// key-value pairs as labels applied to the node.
+	// +optional
+	WebhookNodeSelector map[string]string `json:"webhookNodeSelector,omitempty"`
```

Comment:
I think we should not make this specific to implementation details - us using a webhook. Could we instead have it so there is a selector/toleration for "control plane" or "infra" ... not sure about the naming.

Reply:
I thought about using infra initially but control plane looks like a more inspired choice. Should I replaced Webhook with ControlPlane for nodeselectors & tolerations?

Reply:
I think that would be nice, although I'm little conflicted as everything we have is kind of a control plane, mmm.

Anyhow, I would like to hear @rhrazdil input on this as he worked on similar topic in the past.